docs/streams/upgrade-guide.html

```diff
@@ -85,6 +85,11 @@ <h3><a id="streams_api_changes_220" href="#streams_api_changes_220">Streams API
         used in a try-with-resource statement. For a full list of public interfaces that get impacted please read <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-376%3A+Implement+AutoClosable+on+appropriate+classes+that+want+to+be+used+in+a+try-with-resource+statement">KIP-376</a>.
     </p>
 
+    <p>
```

Comment:
This is under the

```
<h3><a id="streams_api_changes_220" href="#streams_api_changes_220">Streams API changes in 2.2.0</a></h3>
```

Is that intentional?

I think we should create new `Streams API changes in 2.3.0` section on top and add this paragraph there.

Reply:
> I think we should create new `Streams API changes in 2.3.0` section on top and add this paragraph there.

Thanks for reviews. I will address this comment by next commit.